supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,104 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A _doc text_ is a short description of an engineering Bugzilla or Jira issue that is published in product Errata advisories and Release notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with _The IBM Style Guide_. You can write doc texts in more than one way, depending on the issue type: 
+
+* Enhancement
+* Bug Fix
+* Known Issue
+* Technology Preview
+* Deprecated Functionality
+
+Use the following general structure for consistency.
+
+Enhancement::
```

Comment:
So this syntax (Enhancement::) causes the first line after it to be indented, but not others.

I'd either suggest making this a discrete heading:

```
[discrete]
==== Enhancement
```

Or using `+` between each of the items/examples under each type, so that they are indented properly with each type.

```
Enhancement::

Use present tense in one of the following formats:
+
----
This enhancement <present tense explanation>.
----
+
...
```